float64 accel_x
float64 accel_y
float64 accel_z
float64 gyro_x
float64 gyro_y
float64 gyro_z
float64 state
float64 swing_time
float64 t
float64 steps
